code/edgeconsent/src/test/java/com/adobe/marketing/mobile/edge/consent/ConsentExtensionVersionTest.java

```diff
@@ -12,66 +12,13 @@
 package com.adobe.marketing.mobile.edge.consent;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
 
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Properties;
 import org.junit.Test;
 
 public class ConsentExtensionVersionTest {
 
-	private static String GRADLE_PROPERTIES_PATH = "../gradle.properties";
-	private static String PROPERTY_MODULE_VERSION = "moduleVersion";
-
 	@Test
-	public void extensionVersion_verifyModuleVersionInPropertiesFile_asEqual() {
-		Properties properties = loadProperties(GRADLE_PROPERTIES_PATH);
-
-		assertNotNull(Consent.extensionVersion());
-		assertFalse(Consent.extensionVersion().isEmpty());
-
-		String moduleVersion = properties.getProperty(PROPERTY_MODULE_VERSION);
-		assertNotNull(moduleVersion);
-		assertFalse(moduleVersion.isEmpty());
-
-		assertEquals(
-			String.format(
-				"Expected version to match in gradle.properties (%s) and extensionVersion" + " API (%s)",
-				moduleVersion,
-				Consent.extensionVersion()
-			),
-			moduleVersion,
-			Consent.extensionVersion()
-		);
-	}
-
-	// ========================================================================================
-	// Private methods
-	// ========================================================================================
-
-	private Properties loadProperties(final String filepath) {
-		Properties properties = new Properties();
-		InputStream input = null;
-
-		try {
-			input = new FileInputStream(filepath);
-
-			properties.load(input);
-		} catch (IOException e) {
-			e.printStackTrace();
-		} finally {
-			if (input != null) {
-				try {
-					input.close();
-				} catch (IOException e) {
-					e.printStackTrace();
-				}
-			}
-		}
-
-		return properties;
+	public void internalExtensionVersion_publicExtensionVersion_asEqual() {
```

Comment:
nit: This test case name isn't actually accurate. Maybe rename to `extensionVersion_verifyVersionInConstants_asEqual()`